hack/dockerfile/binaries-commits

```diff
@@ -11,5 +11,5 @@ VNDR_COMMIT=c56e082291115e369f77601f9c071dd0b87c7120
 BINDATA_COMMIT=a0ff2567cfb70903282db057e799fd826784d41d
 
 # CLI
-DOCKERCLI_REPO=https://github.com/dperny/cli
-DOCKERCLI_COMMIT=7230906e0e297999eb33da74e0279c5cf41a119e
+DOCKERCLI_REPO=https://github.com/tonistiigi/docker-cli
+DOCKERCLI_COMMIT=79c802d4ca29900ee39a1ff0961c3aca99c7eff3
```

Comment:
This will be removed. I included the cli change so that people testing it with cli do not need to check it out themselves. The change itself is backward compatible with old cli.